projects/aca-content/src/lib/components/search/search-save/list/ui-list/saved-searches-list.ui-component.ts

```diff
@@ -0,0 +1,148 @@
+/*!
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { AfterContentInit, Component, DestroyRef, EventEmitter, inject, Input, Output, ViewEncapsulation } from '@angular/core';
+import {
+  AppConfigService,
+  DataCellEvent,
+  DATATABLE_DIRECTIVES,
+  DataTableComponent,
+  DataTableSchema,
+  ShowHeaderMode,
+  TEMPLATE_DIRECTIVES
+} from '@alfresco/adf-core';
+import { Subject } from 'rxjs';
+import { CommonModule } from '@angular/common';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { SavedSearchesListUiService } from '../saved-searches-list-ui.service';
+import { savedSearchesListSchema } from '../smart-list/saved-searches-list-schema';
+import { SavedSearch } from '@alfresco/adf-content-services';
+import { Clipboard } from '@angular/cdk/clipboard';
+import { Store } from '@ngrx/store';
+import { SnackbarInfoAction } from '@alfresco/aca-shared/store';
+
+@Component({
+  selector: 'aca-saved-searches-ui-list',
+  standalone: true,
+  imports: [CommonModule, DATATABLE_DIRECTIVES, TEMPLATE_DIRECTIVES, DataTableComponent],
+  templateUrl: './saved-searches-list.ui-component.html',
+  styleUrls: ['./saved-searches-list.ui-component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  host: { class: 'aca-saved-searches-ui-list' }
+})
+export class SavedSearchesListUiComponent extends DataTableSchema implements AfterContentInit {
+  @Input()
+  savedSearches: SavedSearch[] = [];
+
+  @Output()
+  savedSearchOrderChanged = new EventEmitter<{ previousIndex: number; currentIndex: number }>();
+
+  readonly ShowHeaderMode = ShowHeaderMode;
+
+  private readonly savedSearchesListUiService = inject(SavedSearchesListUiService);
+  private readonly destroyRef = inject(DestroyRef);
```

Comment:
That will work only in injection context, if you want to use `takeUntilDestroyed` outside on injection context you have to provide `DestroyRef` otherwise you'll get an error saying that you can't run it outside of injection context

Reply:
ah ok, good to know, thanks! :)